packages/flutter/lib/src/material/stepper.dart

```diff
@@ -122,6 +122,57 @@ class Step {
 /// to this widget based on some logic triggered by the three callbacks that it
 /// provides.
 ///
+/// {@tool sample --template=stateful_widget_scaffold_center}
+///
+/// ```dart
+/// int _index = 0;
+/// Widget build(BuildContext context) {
+///   return Container(
+///     height: 300,
+///     width: 300,
+///     child: Stepper(
+///       currentStep: _index,
+///       onStepCancel: () {
+///         if (_index <= 0) {
+///          return null;
+///         }
+///         setState(() {
+///           _index--;
+///         });
+///       },
+///       onStepContinue: () {
+///         if (_index >= 1) {
+///          return null;
```

Comment:
These callbacks have a `void` return type, so it's not necessary to return anything.

```suggestion
///          return;
```